pom.xml

```diff
@@ -112,6 +112,22 @@
             <artifactId>spring-boot-starter-websocket</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka</artifactId>
```

Comment:
what version? We stick to particular kafka version across vnoc

Reply:
Library version does not need to be a 100% match by pulling vnoc platform here I think, but it would make sense to specify a current version so that majors match at least

Reply:
90% match 😄

Reply:
added specific version